corehq/ex-submodules/phonelog/models.py

```diff
@@ -30,6 +31,34 @@ def __repr__(self):
         return u"DeviceReportEntry(domain='{}', msg='{}')".format(self.domain, self.msg)
 
 
+@architect.install('partition', type='range', subtype='date', constraint='day', column='server_date')
+class PartitionedDeviceReportEntry(models.Model):
```

Comment:
This is probably confusing naming since it's not using our partitions in the same way as forms/cases/sms. Might want to change it if you guys want to go through with this change